Automattic/wp-calypso
# Email Forwarding: Fix Incorrect Extended Limit

Fixes #82812

## Proposed Changes

* Fix the incorrect Email Forwarding limit preventing users on the Creator plan from adding more email forwards (they can still do so with the direct URL; it's just an interface problem)

## Testing Instructions

* Navigate to My Sites > Upgrades > Email on a site with 25 email forwards (or force this to always appear)
* Confirm that on a site below the Creator plan, the same 25 limit appears, but that limit increases to 100 for sites with the Creator plan.

<img width="1054" alt="Screenshot 2024-01-07 at 10 23 16" src="https://github.com/Automattic/wp-calypso/assets/43215253/c8fbf097-fb30-49fd-8e8f-c0e7d567ab2b">

**Please note:** I do not have a Creator plan with a custom domain that I can test this with. I've only tested it by switching features, but `FEATURE_EMAIL_FORWARDING_EXTENDED_LIMIT` looks to be correctly registered to the Creator plan here so I'm pretty sure this will work. 

https://github.com/Automattic/wp-calypso/blob/b3d3b2cddb177801bff78d165b3deab5e2a47431/packages/calypso-products/src/plans-list.tsx#L1145

cc @ebinnion @wongasy - seems 100 was referenced in #81264 but not included?
